tests/foreman/ui/test_discoveredhost.py

```diff
@@ -950,3 +950,224 @@ def test_negative_pxe_less_discovery_without_dhcp(self):
 
         @CaseLevel: System
         """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_associate_org_loc_from_new_model_window(self):
+        """Provision a discovered host manually by associating org & loc from
+        host properties model window and select create host button.
+
+        @id: 8c6a7d3f-e34e-4888-9b1c-58e71ee584a3
+
+        @Assert: Provisioned host is associated with selected org & location
+
+        @caseautomation: notautomated
+
+        @CaseLevel: System
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_associate_hostgroup_from_new_model_window(self):
+        """Provision a discovered host manually by associating hostgroup from
+        host properties model window and select create host button.
+
+        @id: f17fb8c9-f9cb-4547-80bc-3b40c6691bb1
+
+        @Assert: Provisioned host is created with selected host-group
+
+        @caseautomation: notautomated
+
+        @CaseLevel: System
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_using_quick_host_button(self):
+        """Associate hostgroup while provisioning a discovered host from
+        host properties model window and select quick host.
+
+        @id: 34c1e9ea-f210-4a1e-aead-421eb962643b
+
+        @Setup:
+
+        1. Host should already be discovered
+        2. Hostgroup should already be created with all required entities.
+
+        @Assert: Host should be quickly provisionioned.
+
+        @caseautomation: notautomated
+
+        @CaseLevel: System
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_keep_facts_set_by_user(self):
```

Comment:
Basically main action of the test is missing in the test name. How about test_positive_*provision*_with_facts_set_by_user?